kotlinx-coroutines-core/common/src/Job.kt

```diff
@@ -117,6 +117,22 @@ public interface Job : CoroutineContext.Element {
 
     // ------------ state query ------------
 
+    /**
+     * Returns a parent of the current job if the parent-child relationship
```

Comment:
```suggestion
     * Returns the parent of the current job if the parent-child relationship
```
IIRC, the parent, if it exists, is unique?